### DISCUSSION TITLE
Build error occurred

### DISCUSSION THREAD

**Discussion by lalidiaz** [vercel/next.js#17603] [Help]
Hi everyone!
I am newbie.... I am trying to deploy my Nextjs app but everytime I try, gives me a build error! 
This is the error, I really don't know what to do... I've been trying to solve this for a couple of days already but nothing seems to be working :( 

![Screen Shot 2020-10-04 at 7 02 52 PM](https://user-images.githubusercontent.com/60779542/95019040-3760dd00-0674-11eb-9fa7-ebc4fe3f3377.png)

**Comment by c0b41** [comment]
can you show what do you have  `/pages/status/[id].js`

**Reply by lalidiaz** [reply]
Yes, here it is :)

![Screen Shot 2020-10-04 at 7 15 27 PM](https://user-images.githubusercontent.com/60779542/95019373-0f727900-0676-11eb-8ca1-cc550bddbd12.png)
![Screen Shot 2020-10-04 at 7 15 34 PM](https://user-images.githubusercontent.com/60779542/95019374-113c3c80-0676-11eb-8605-f3c351b39b75.png)

**Reply by c0b41** [reply]
probably firestore return nothing check if data have something

**Reply by c0b41** [reply]
also do not pass date object ref https://nextjs.org/docs/basic-features/data-fetching#getstaticprops-static-generation

**Reply by lalidiaz** [reply]
Thank you for the response! 
I check data and is returning an object... so firebase is okey I think... 
Regarding the date object... I am not sure if I understood well... do you mean I have to delete "{createdAt} = data"?

**Reply by c0b41** [reply]
you can't pass props to date object this need to be string. ref https://github.com/vercel/next.js/discussions/11498

**Reply by lalidiaz** [reply]
I will try to check it now! Thank you 👍
